ci/builders/linux_android_emulator.json

```diff
@@ -72,9 +72,51 @@
                     "contexts": [
                         "android_virtual_device"
                     ],
-                    "script": "flutter/testing/scenario_app/run_android_tests.sh",
+                    "script": "flutter/testing/scenario_app/run_android_tests.sh --graphics-backend=skia",
```

Comment:
Is there a reason that `--graphics-backend=skia` is in the script string and not the parameters list below? Like, is this working around a bug in the engine_v2 recipe?

Reply:
Looks like the presub is failing because the flag does indeed need to go in the parameter list rather than in the script string.